vars/perfRegressionParallelPipeline.groovy

```diff
@@ -45,62 +54,122 @@ def call(Map pipelineParams) {
             buildDiscarder(logRotator(numToKeepStr: '20'))
         }
         stages {
-            stage('Run SCT Test') {
+            stage('Run SCT Performance Tests') {
                 steps {
-                    for (sub_test in pipelineParams.sub_tests) {
-                        def current_sub_test = sub_test;
-                        tasks["sub_test=${current_sub_test}"] = {
-                            node(getJenkinsLabels(params.backend, params.aws_region)) {
-                                withEnv(["AWS_ACCESS_KEY_ID=${env.AWS_ACCESS_KEY_ID}",
-                                             "AWS_SECRET_ACCESS_KEY=${env.AWS_SECRET_ACCESS_KEY}",]) {
-
-                                    wrap([$class: 'BuildUser']) {
-                                        dir('scylla-cluster-tests') {
-                                            checkout scm
-
-                                            dir("scylla-qa-internal") {
-                                                git(url: 'git@github.com:scylladb/scylla-qa-internal.git',
-                                                    credentialsId:'b8a774da-0e46-4c91-9f74-09caebaea261',
-                                                    branch: 'master')
+                    script {
+                        def tasks = [:]
+                        def sub_tests = new JsonSlurper().parseText(params.sub_tests)
+                        for (sub_test in sub_tests) {
+                            def current_sub_test = sub_test
+                            tasks["sub_test=${current_sub_test}"] = {
+                                node(getJenkinsLabels(params.backend, params.aws_region)) {
+                                    withEnv(["AWS_ACCESS_KEY_ID=${env.AWS_ACCESS_KEY_ID}",
+                                                 "AWS_SECRET_ACCESS_KEY=${env.AWS_SECRET_ACCESS_KEY}",]) {
+                                        stage("${current_sub_test}"){
+                                            catchError(stageResult: 'FAILURE') {
+                                                wrap([$class: 'BuildUser']) {
+                                                    dir('scylla-cluster-tests') {
+                                                        checkout scm
+
+                                                        dir("scylla-qa-internal") {
```

Comment:
why do we need here scylla-qa-internal?
We may need it in the future (and needed in the past) but for now, It's not needed.

Reply:
removed